marketplace-service/src/main/java/com/axonivy/market/constants/RequestMappingConstants.java

```diff
@@ -13,6 +13,7 @@ public class RequestMappingConstants {
   public static final String FEEDBACK = API + "/feedback";
   public static final String IMAGE = API + "/image";
   public static final String SYNC = "sync";
+  public static final String SYNC_ONE_PRODUCT = "sync/product";
```

Comment:
The URL like : http://localhost:8080/api/product/sync/product?id=skribble-connector&marketItemPath=market%2Fconnector%2Fskribble-connector%2F

I see it duplicated "product". Should we only use http://localhost:8080/api/product/sync?id=skribble-connector&marketItemPath=market%2Fconnector%2Fskribble-connector%2F